### PR TITLE
feat(3076582507): Alert component

### PR DESCRIPTION
**Pull request by dan-ziv**
### Description of the Changes

Adding a new and generic `Alert` component and replacing all the custom messages with this component.

Solves #[3076582507](https://starkware.monday.com/boards/2847524005/pulses/3076582507)
***
### Checklist
- [x] Changes have been done against `dev` branch, and PR does not conflict.
- [x] Manually tests of the main Application flows are done and passed.
- [ ] New unit / functional tests have been added (whenever applicable).
- [x] Automation tests are passing in local environment.
- [ ] Docs have been updated (whenever relevant).
- [x] PR title is follow the [Conventional Commits](https://www.conventionalcommits.org/) convention: `<type>[optional scope]: <description>`, e.g: `fix: prevent racing of requests`

- - -
This change is [Reviewable](https://reviewable.io/reviews/starkware-libs/starkgate-frontend/272)

### CODE REVIEW

File: src/components/UI/Alert/Alert.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      default:
+        return <InfoIcon />;
+    }
+  };
+
+  return (
+    <div className={toClasses(styles.alert, styles[type])}>
+      <div className={styles.icon}>{renderIcon()}</div>
+      <div className={styles.text}>
+        {title && <div className={styles.title} dangerouslySetInnerHTML={{__html: title}}></div>}
```
Comment (galuzan-starkware) [inline src/components/UI/Alert/Alert.js:35]: @dan-ziv Can you please elaborate on why you decided to use `dangerouslySetInnerHTML`?

**Review by galuzan-starkware — APPROVED** [review]

**Comment by galuzan-starkware** [comment]
image
image

Not the same styling as the design (for example icon should be more to the left)

**Comment by dan-ziv** [comment]
@galuzan-starkware fixed

**Pull request merged**